src/plugins/janus_sip.c

```diff
@@ -7392,6 +7401,60 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
 	return NULL;
 }
 
+/* Check peer RTP has RFC2833 and push event */
+static void janus_sip_check_rfc2833(janus_sip_session *session, char *buffer, int len) {
+	janus_rtp_header *rtp_header = (janus_rtp_header *)buffer;
+	if (rtp_header->type != JANUS_RTP_RFC2833_DEFAULT_PAYLOAD_TYPE)
+		return;
+	int plen = 0;
+	char *payload_buffer = janus_rtp_payload(buffer, len, &plen);
+	int rfc2833_payload_size = sizeof(janus_rtp_rfc2833_payload);
+	if(plen < rfc2833_payload_size)
```

Comment:
Why not simply do this?

    if(plen < sizeof(janus_rtp_rfc2833_payload))

I don't think there's any need to assign the value to a new `rfc2833_payload_size` variable, since we don't use it anywhere else.